src/hotspot/os/linux/mallocInfoDcmd.cpp

```diff
@@ -36,15 +37,15 @@ void MallocInfoDcmd::execute(DCmdSource source, TRAPS) {
 #ifdef __GLIBC__
   char* buf;
   size_t size;
-  ALLOW_C_FUNCTION(::open_memstream, FILE* stream = ::open_memstream(&buf, &size);)
+  FILE* stream = ::open_memstream(&buf, &size);
```

Comment:
Note that open_memstream was never forbidden, so the ALLOW_C_FUNCTION was not needed.